File: Data Structures & Algorithms/top-k-elements-in-list/submission-0.cpp
class Solution {
public:
    vector<int> topKFrequent(vector<int>& nums, int k) {
        unordered_map<int,int> mp;

        for(int el : nums){
            mp[el]++;
        }
        vector<pair<int,int>> v;
        for(auto el : mp){
            v.push_back(el);
        }
        sort(v.begin(), v.end(), [](pair<int,int>a, pair<int,int>b){
            return a.second>b.second;
            });

        vector<int> ans;
        int i=0;
        while(k--){
            ans.push_back(v[i].first);
            i++;
        }
        return ans;
    }
};

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int i = 0, j = nums.length-1;
        int n = nums.length;

        while(i<j){
            if(nums[i] + nums[j] == target){
                break;
            } else if (nums[i] + nums[j] > target){
                j--;
            } else {
                i++;
            }
        }

        int[] ans = {i+1, j+1};
        return ans;
    }
}
